app/src/main/java/com/alphawallet/app/ui/TokenManagementActivity.java

```diff
@@ -64,15 +91,42 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         hideZeroBalanceCheckBox.setOnCheckedChangeListener((v, checked) -> {
             pref.edit().putBoolean("hide_zero_balance_tokens", checked).apply();
         });
```

Comment:
what does this do? Can you add a comment why we need the jake wharton library here?

Can we achieve the same thing without it using Runnable?

Reply:
The RX Binding library binds Search EditText directly and apply filterable to it. Hence we used it inside the code. Definitely we can remove the overburden of library and implement ourselves.